May2016Scans_Configs/vdmDriverII_Config_PCC_4954.json

```diff
@@ -125,14 +130,16 @@
 	 {
 	 "Comment": "Input graphs file in <AnalysisDir>/<Luminometer>/graphs", 
 	 "InputGraphsFile" : "graphs/graphs_4954_noCorr.pkl",
-	 
-   "Comment": "This is for SIM fit--not used because over-written below.",
+	 "InputGraphsFile" : "graphs/graphs_4954_LengthScale.pkl",
+	 "InputGraphsFile" : "graphs/graphs_4954_LengthScale_BeamBeam.pkl",
+	 "InputGraphsFile" : "graphs/graphs_4954_BeamBeam.pkl",
+	 "InputGraphsFile" : "graphs/graphs_4954_BeamBeam_LengthScale.pkl",
+   
    "FitName" : "SimCapSigma_PCCAndVtx",
-	 "FitConfigFile" : "fits/SimCapSigma_PCCAndVtx_Config.json",
-	 "PlotsTempPath": [["./plotstmp/PCC/"], ["./plotstmp/VTX/"]]
-	 
-   "Comment": "This is for stand-alone PCC--overwrites above",
+   "FitPartner" : "VTX",
```

Comment:
This is a goof ad-on but the code in vdmDriverII & vdmFiterII has to be changed accordingly